### DISCUSSION TITLE
Should we have a social media presence?

### DISCUSSION THREAD

**Discussion by Pheogrammer** [Laravel-Lang/common#183] [Ideas]
Was just thinking after having a success message in testing one of my pipelines.. 

Was wondering if we can have dedicated social media for maybe telling the world about our work here with Laravel Lang.. that way we can have more active users, contributors (and new contents maybe)

...just thinking out loud

**Comment by andrey-helldar** [comment]
It's a good idea, but I can't do it alone.
Right now, we have the following channels of communication:

- https://laravel-lang.com
- https://boosty.to/laravel-lang
- https://t.me/LaravelLangEng
- https://t.me/LaravelLangRus

**Comment by Pheogrammer** [comment]
I can help with that.


The current channels i think they are more based for developers and people who know about our product and stuff like that.

To gain more traction we need something more open and explorable through social media like LinkedIn and  Twitter...

**Reply by andrey-helldar** [reply]
I can still set up Twitter to publish releases, but I have a problem with LinkedIn. I'm in Russia and this service won't even let me delete my account, as they won't send an SMS with a code to a Russian number.
But, having an access token for publishing news about releases, I can do it.

Of course, if we're talking about automatic messages.

**Reply by andrey-helldar** [reply]
I return from vacation in a week and will be able to send the original logo and splash screen files. They are on my computer at home. You can use them if you need to.

**Reply by Pheogrammer** [reply]
That's bad, 

But I remember coming through these virtual numbers that you can use to receive messages.. you can try using that

**Reply by Pheogrammer** [reply]
Okay, happy vacation

**Reply by andrey-helldar** [reply]
By the way, I just remembered one peculiarity - due to the technical impossibility of paying for DigitalOcean services, I moved my web server to a Russian data center where I can pay for it. The webhook handler for Laravel Lang is located on it.

And the problem is that without a VPN, access to Twitter and LinkedIn is impossible. Access to Twitter is restricted by the Russian government, and LinkedIn themselves have banned access to them.

So there are also problems with automatic publication of release information.

While the government solves its problems, ordinary people suffer.

**Comment by Pheogrammer** [comment]
Well, we can simply do the following:

- I will handle the posting of the updates manually to the social medias, 
- I will make sure to create the posts/images the day I know there will be an update
- We will go through the contents to be posted before i manually do.

How about that?

**Reply by andrey-helldar** [reply]
I think it's possible. We haven't had many changes in the last few months, so I've been publishing releases as soon as they come out. For automatic publishing there is no problem, but if you do it manually...
So it was decided to go back to the previous "version" of publishing - releasing releases every Tuesday once a week.

I think in this case it would be possible to manually report it.

You could also discuss the approximate time at which new versions will be released, so that you don't have to constantly check the repository several times a day. Or, even better, when a new version is released I can leave a trigger message. So, GitHub will send a notification about it to the mail if other notifications were turned off.

**Reply by Pheogrammer** [reply]
That could work... let's say we have different posts apart from releases.. we also publish _How to_ do anything & everything with Laravel lang

**Reply by andrey-helldar** [reply]
In principle, you can. At the moment we don't have many projects that fit under the "How to" section, but I'm trying to find time to improve the publisher and also do the two projects that are in the private section :)

**Comment by andrey-helldar** [comment]
@Pheogrammer, hi! You can take images from our new public repository: https://github.com/Laravel-Lang/art

**Reply by Pheogrammer** [reply]
> @Pheogrammer, hi! You can take images from our new public repository: https://github.com/Laravel-Lang/art

Sure, I will check it

**Reply by andrey-helldar** [reply]
You can take any SVG file as a source image and create a new one with it.

You can also add new images to this repository if you want.
